proto/process/agent.proto

```diff
@@ -1665,3 +1665,32 @@ message AgentConfiguration {
 message DNSDatabaseEntry {
 	repeated int32 nameOffsets = 1;
 }
+
+message VerticalPodAutoscaler {
+	Metadata metadata = 1;
+	VerticalPodAutoscalerSpec spec = 2;
+	VerticalPodAutoscalerStatus status = 3;
+}
+
+message VerticalPodAutoscalerSpec {
+	VerticalPodAutoscalerTarget target = 1;
+	string updateMode = 2; // Extracted from UpdatePolicy
+}
+
+message VerticalPodAutoscalerTarget{
+	string kind = 1;
+	string name = 2;
+}
+
+message VerticalPodAutoscalerStatus {
+	int64 LastRecommendedDate = 1; // Extracted from Codition=RecommendationProvided AND Status=true
+	repeated ContainerRecommendations recommendations = 2;
+}
+
+message ContainerRecommendations {
+	string containerName = 1;
+	string target = 2;
```

Comment:
How do you differentiate the CPU and Memory ones?

Reply:
ie
```
      Target:
        Cpu:     43
        Memory:  262144k
```

Also it should probably be a `int64` like we do for request/limits (with the nanocore and bytes) https://github.com/DataDog/agent-payload/blob/2ab4977ccd9f78c28e8a1e04812b74a6493e0a7d/proto/process/agent.proto#L1118

Reply:
I found https://github.com/DataDog/agent-payload/blob/jlineaweaver/vpa/proto/process/agent.proto#L1140-L1143. This seems to map correctly.  For example 
```
Lower Bound:
        Cpu:     1
        Memory:  5526504030
Target:
        Cpu:     1
        Memory:  5527335984
 Uncapped Target:
        Cpu:     1
        Memory:  5527335984
```